lizaku/assignment1/Q1.py

```diff
@@ -0,0 +1,49 @@
+import string
+import re
+
+def anagrams_words(w1, w2, case_insensitive=True):
+    if case_insensitive:
+        w1 = w1.lower()
+        w2 = w2.lower()
+    if len(w1) != len(w2):
+        return False
+    letters1 = list(w1)
+    letters2 = list(w2)
+    while letters1 != [] or letters2 != []:
+        for letter in letters1:
+            if letter in letters2:
+                letters1.remove(letter)
```

Comment:
Please explain in a comment why you are removing letters from the list after finding them :)